dev-util/cppunit/cppunit-1.14.0.recipe

```diff
@@ -50,6 +55,9 @@ BUILD_PREREQUIRES="
 	cmd:sed
 	"
 
+defineDebugInfoPackage cppunit$secondaryArchSuffix \
+	$libDir/libcppunit-$portVers.so.$libVersion
+
 BUILD()
 {
 	libtoolize --force --copy --install
```

Comment:
`autoreconf` would call `libtoolize` automatically if needed, so this line is not required

```diff
-	libtoolize --force --copy --install
```